evap/grades/views.py

```diff
@@ -165,6 +165,8 @@ def edit_grades(request, semester_id, course_id, grade_document_id):
 
     form = GradeDocumentForm(request.POST or None, request.FILES or None, instance=grade_document)
 
+    final_grades = request.GET.get("final") == "true"  # if parameter is not given, assume midterm grades
```

Comment:
This could then become
```
final_grades = grade_document.type == GradeDocument.Type.FINAL_GRADES
```
which would properly utilize the constant enum-like names we have for the two types without relying on whatever string is used in the database.